src/webgl/functions.js

```diff
@@ -0,0 +1,149 @@
+/* eslint-disable */
```

Comment:
>  clear() and readPixels() function part of some objects instead of standalone functions on a global objects?

I like this idea, I also think they look a little odd here. The question is just "which object"?

I have started to think that `draw` should probably be a method on `Program` - it runs the shaders in the program selected with `Program.use`

`clear` and `readPixels` seem harder to associate with any existing object. I believe that they work on the currently selected framebuffer - and we don't have an object for the global framebuffer.

I suppose they could be static methods on `Framebuffer`...